src/en/tut-lxd.md

```diff
@@ -0,0 +1,264 @@
+Title: Getting started with Juju and LXD
+
+# Getting started with Juju and LXD
+
+[LXD](lxd-upstream) provides an amazing experience using machine containers to be able to
```

Comment:
This first sentence seems to conflate a number of things and is difficult to grasp. I would suggest something like 

You may already know that [LXD] provides an amazing experience for managing lightweight containers. Juju can make use of LXD to provide a cloud-like experience from your local machine - etc etc


Possibly a short bullet list of why you would use it. We don't want to be too wordy in the beginning